app/classes/Transvision/ShowResults.php

```diff
@@ -479,7 +488,15 @@ public static function resultsTable($search_object, $search_results, $page)
                     <div dir='ltr' class='result_meta_link'>
                       <a class='source_link' href='{$locale2_path}'>
                         &lt;source&gt;
-                      </a>
+                      </a>";
+            if (isset($edit_link)) {
+                $table .= "
+                            &nbsp;
+                            <a class='edit_link' target='_blank' href='{$edit_link}'>
+                            &lt;edit&gt;
```

Comment:
If we don't want to put a tooltip, given it's bad for accessibility, I would make this link "<edit in Pootle>" and "<edit in Pontoon>", depending on the tool. You can set the text in the if cycle where you create the link.